lib/util.js

```diff
@@ -148,6 +148,27 @@ util = {
         return result;
     },
 
+    /**
+     * A helper function to determine if the provided v1 entity has legacy properties.
+     *
+     * @private
+     * @param {Object} entityV1 - The v1 entity to be checked for the presence of legacy properties.
+     * @param {String} type - The type of property to be adjudged against.
+     * @returns {Boolean} - A flag to indicate the legacy property status of the passed v1 entity.
+     */
+    notLegacy: function (entityV1, type) {
+        if (!entityV1) { return; }
+
+        switch (type) {
+            case 'script':
+                return !(entityV1.tests || entityV1.preRequestScript);
+            case 'auth':
+                return !entityV1.currentHelper || _.isEmpty(entityV1.helperAttributes);
```

Comment:
should be `_.isNil(entityV1.helperAttributes)`, `{ currentHelper: 'basicAuth', helperAttributes: {} }` is a valid legacy auth definition